test/esinstall/__snapshots__/install.test.js.snap

```diff
@@ -1,5 +1,39 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`snowpack install auto-named-exports: allFiles 1`] = `
+Array [
+  "cjs-named-export-pkg-02.js",
+  "import-map.json",
+]
+`;
+
+exports[`snowpack install auto-named-exports: cjs-named-export-pkg-02.js 1`] = `
+"var export1 = 'foo';
+var export2 = 'bar';
+var entrypoint = {
+	export1: export1,
+	export2: export2
+};
+export default entrypoint;
+export { entrypoint as __moduleExports, export1, export2 };"
```

Comment:
Interesting! It’s fun to see things progress to interop better between CJS and ESM